src/GameObjects/Player.py

```diff
@@ -1,17 +1,26 @@
 import pygame
 
-from Interface.GameObjectInterface import GameObject
+from GameObjects.Interface.GameObjectInterface import GameObject
 
 
 class Player(GameObject):
-    def __init__(self, x, y):
+    def __init__(self, x, y, width, height):
         self.x = x
         self.y = y
+        self.width = width
+        self.height = height
         # Modify the hitbox size depending on the Image size for player
-        self.hitbox = (self.x, self.y, 20, 20)
+        self.hitbox = (self.x, self.y,self.width, self.height)
 
     def drawCharacter(self, canvas):
-        pass
+        pygame.draw.rect(canvas,(0,255,0),self.hitbox)
+        self.hitbox = (self.x, self.y, self.width, self.height)
+
+    def propagate(self, canvas, step):
+        self.drawCharacter(canvas)
```

Comment:
I don't see a need to call self.drawCharacter from here.
We can always do that from the Game file